src/index.json

```diff
@@ -46,6 +46,62 @@
                 "sha256Digest": "4ac7b8a4a89eda68d9d1a07cc5edd9b1a2b88421e2aa9a9e5b86a241f127775f"
             }
         ],
+        "ai-examples": [
+            {
+                "downloadUrl": "https://aladdinprodwestus.blob.core.windows.net/aladdin-extensions/ai_examples-0.1.0-py2.py3-none-any.whl",
+                "filename": "ai_examples-0.1.0-py2.py3-none-any.whl",
+                "metadata": {
+                    "azext.isPreview": true,
+                    "azext.maxCliCoreVersion": "2.1.0",
+                    "azext.minCliCoreVersion": "2.0.65",
+                    "classifiers": [
+                        "Development Status :: 4 - Beta",
+                        "Intended Audience :: Developers",
+                        "Intended Audience :: System Administrators",
+                        "Programming Language :: Python",
+                        "Programming Language :: Python :: 2",
+                        "Programming Language :: Python :: 2.7",
+                        "Programming Language :: Python :: 3",
+                        "Programming Language :: Python :: 3.4",
+                        "Programming Language :: Python :: 3.5",
+                        "Programming Language :: Python :: 3.6",
+                        "License :: OSI Approved :: MIT License"
+                    ],
+                    "extensions": {
+                        "python.details": {
+                            "contacts": [
+                                {
+                                    "email": "mabooe@microsoft.com",
+                                    "name": "Matthew Booe",
+                                    "role": "author"
+                                }
+                            ],
+                            "document_names": {
+                                "description": "DESCRIPTION.rst"
+                            },
+                            "project_urls": {
+                                "Home": "https://github.com/Azure/azure-cli-extensions/tree/master/src/ai-examples"
+                            }
+                        }
+                    },
+                    "extras": [],
+                    "generator": "bdist_wheel (0.30.0)",
+                    "license": "MIT",
+                    "metadata_version": "2.0",
+                    "name": "ai-examples",
+                    "run_requires": [
+                        {
+                            "requires": [
+                                "azure-cli-core"
```

Comment:
No need to explicitly to specify `azure-cli-core` I think, azure-cli-extensions depends on azure-cli, and azure-cli depends on azure-cli-core.

You can have a try cause it failed CI.

Reply:
OK, removing the `azure-cli-core` dependency.